workers/data_refinery_workers/processors/smasher_email.html

```diff
@@ -0,0 +1,1501 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Not sure whether you saw my previous comment. So I am posting it here again:
Here is how this web page looks like:
![screenshot from 2018-08-24 12-00-47](https://user-images.githubusercontent.com/15133253/44595084-08d65f80-a796-11e8-8fb6-df817c3b70b7.png)
It looks very simple. Are you sure you need all those CSS lines and tables in the code?

Reply:
If Deepa is using some tools to generate this page automatically, then I guess we don't need to review the HTML code at all. Am I right?

Reply:
I think so, still I'm looking into the logo and the button not being centered. 

Should we minify the code of the template??